src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -56,4 +61,24 @@ void setConversationStore(ConversationStore conversationStore) {
     void setMessageStore(MessageStore messageStore) {
         this.messageStore = messageStore;
     }
+
+    /**
```

Comment:
this file's indents are all +4 spaces whereas other java files are +2. Can you fix for consistency?

Reply:
Done